migrations/020.do.round-to-ie-mapping.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE spark_rounds ADD COLUMN meridian_address TEXT NOT NULL DEFAULT '0x00';
+ALTER TABLE spark_rounds ADD COLUMN meridian_round BIGINT NOT NULL DEFAULT 0;
```

Comment:
Hmm, this is an expensive migration. Our deployment pipeline is timing out.

Reverting this PR in https://github.com/filecoin-station/spark-api/pull/102